website/docs/opengl/1-hello-window.md

```diff
@@ -0,0 +1,263 @@
+{
+    "TableOfContents": {
+        "Name": "1.1 - Hello Window",
+        "Url": "1-hello-window.html",
+        "Metadata": {
+            "AuthorGitHub": "ohtrobinson",
+            "DateTimeWritten": "02/11/2022 12:00",
+            "PreviewImage": ""
+        }
+    }
+}
+---
+
+# 1.1 - Hello Window
+> [!Note]
+> You can view the source code for this tutorial [here.](https://github.com/dotnet/Silk.NET/blob/main/examples/CSharp/OpenGL%20Tutorials/Tutorial%201.1%20-%20Hello%20Window/Program.cs)
+
+## Introduction
+Welcome to your first Silk.NET tutorial!
+
+The first part of your journey will be learning how to create a window and accept some input.
+
+## A basic project
+Okay, let's get started!
+
+The first thing we will need is a blank C# project. For this tutorial, we're going to be using .NET 6, however Silk works with any .NET Standard 2.0 platform.
+
+We're going to assume you know how to create projects with your chosen code editor, so create a blank console project. If you don't know how to do this, here are some good guides to get started:
+* [Creating a basic project using the dotnet cli](https://dotnet.microsoft.com/en-us/learn/dotnet/hello-world-tutorial/intro)
+* [Using Visual Studio to create a .NET project](https://docs.microsoft.com/en-us/visualstudio/get-started/csharp/tutorial-console?view=vs-2022)
+* [Creating and opening projects with Jetbrains Rider](https://www.jetbrains.com/help/rider/Creating_and_Opening_Projects_and_Solutions.html#trusted-and-untrusted-solutions)
+
+If you've used, or are using, an older version of .NET, you will be familliar with the standard C# entry point. .NET 6 changes this by default, however Silk works best using the old format. If you're using an older version of .NET, you can skip this bit.
+
+Add the following code to your file:
+
+```cs
+// If you're using an earlier version of .NET which doesn't have file-scoped namespaces (.NET 5 or earlier), you'll need to encase your class inside the namespace.
+namespace MySilkProgram;
+
+public class Program
+{
+    public static void Main(string[] args) { }
+}
+```
+
+## Installing Silk.NET
+Before you can get started using Silk.NET, you'll want to install Silk.NET. For the purposes of this tutorial, we're going to assume you're already familliar with creating a C# project, as well as how to install packages with your chosen editor.
+
+While there are many packages available, we'll only need three for this tutorial.
+* [Silk.NET.Windowing](https://www.nuget.org/packages/Silk.NET.Windowing)
+* [Silk.NET.Input](https://www.nuget.org/packages/Silk.NET.Input)
+* [Silk.NET.OpenGL](https://www.nuget.org/packages/Silk.NET.OpenGL)
+
+We won't be using the `OpenGL` package in this tutorial, however we'll be using it in every other tutorial, so it's a good idea to install it now.
+
+Using the dotnet cli, installing the packages is simple:
+
+```
+dotnet add package Silk.NET.Windowing
+dotnet add package Silk.NET.Input
+dotnet add package Silk.NET.OpenGL
+```
```

Comment:
Or if you're using Visual Studio, you can use the NuGet Package Manager by right-clicking on your project in the Solution Explorer and selecting "Manage NuGet Packages..."

I think this would be a good addition to the tutorial given that a lot of people use Visual Studio for C#

Reply:
Sounds good, I'll add that - I did mean to add it but I haven't used VS for nearly 2 years so couldn't remember..